services/centralized-kubecost/0.27.0/post-install-jobs/post-install-jobs.yaml

```diff
@@ -42,7 +42,7 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: kubectl
-          image: bitnami/kubectl:1.24.1
+          image: bitnami/kubectl:1.24.6
```

Comment:
Should we bump some of these applications to avoid any auto upgrade issues? Though it might not matter for most of these jobs

Reply:
yeah I meant to do a diff of versions from 2.3, I'll check later and bump the apps that need a version bump

Reply:
hoping to bump kubecost before the release https://github.com/mesosphere/charts/pull/1364